openhands_cli/agent_chat.py

```diff
@@ -110,20 +115,31 @@ def run_cli_entry(
         print_formatted_text(HTML("\n<yellow>Goodbye! 👋</yellow>"))
         return
 
+    console = Console()
+    compat_result = check_terminal_compatibility(console=console)
+    if not compat_result.compatible:
+        message = (
+            "OpenHands CLI terminal UI may not work correctly in this environment: "
+            f"{compat_result.reason}"
+        )
+        if compat_result.is_tty:
+            console.print(message, style="yellow")
+        else:
+            print(message)
+
+        if strict_mode_enabled():
+            sys.exit(2)
```

Comment:
It seems like this forced the app to exit if true

could we make it so that we're proceeding without exiting in the event we're not `compat_result.compatible`